internal/blocktx/processor.go

```diff
@@ -99,15 +107,15 @@ func (p *Processor) Start() error {
 		return nil
 	})
 	if err != nil {
-		return fmt.Errorf("failed to subscribe to %s topic: %w", RegisterTxTopic, err)
+		return errors.Join(ErrFailedToSubscribeToRegisterTopic, err)
```

Comment:
Following the same pattern this could be

```go
errors.Join(ErrFailedToSubscribeToRegisterTopic, fmt.Errorf("topic: %s", RegisterTxTopic), err)
```